wies/projects/templates/parts/placement_table.html

```diff
@@ -1,70 +1,74 @@
-<div class="rvo-table--responsive">
-  <table class="rvo-table" id="placement_table">
+<div class="rvo-table--responsive" id="placement_table">
```

Comment:
the clickable area to expand the filters is a lot larger than just the button. i accidentally opened it and got confused on what happened (also happens on opdrachten)

https://github.com/user-attachments/assets/a9187a11-01a5-4da0-b1a0-d18786d22d3b